recipe/meta.yaml

```diff
@@ -34,6 +34,7 @@ requirements:
     - graphite2
   run:
     - cairo
+    - libglib
```

Comment:
This is redundant. `glib` has a `run_exports` for `libglib` and thus this should be added automatically by conda-build.